### DISCUSSION TITLE
An option to only import visible objects in .blend files

### DISCUSSION THREAD

**Discussion by TumGuy** [godotengine/godot-proposals#5331] [3D]
I am in awe of the 4.0 Blender support and the ability to bring .blend files into your Godot scene, but there's one feature not present which I would love to see. In my current workflow from Blender to Godot 3.5, I use the export add-on for Blender which creates an .escn file for Godot. I tend to have reference objects that I will disable the visibility of within Blender, and then set the .escn to only export visible objects. However when I try to open a .blend file in Godot 4.0, every object in my Blender hierarchy is visible and I have to select each node and select the "Skip import" option. 

Some of my .blend files have up to a dozen or so references that I use while building so it would save a lot of tedious clicking to have the ability to mass skip import on objects that are invisible in the Blender hierarchy, just like in the Godot export add-on for Blender.

**Comment by RedMser** [comment]
As a workaround, you can create an import script which hides objects based on some criteria like name or type. But I agree this is something we definitely need.

**Comment by jtnicholl** [comment]
It already has this option.
Advanced importer window -> Scene on left panel -> Blender on right panel -> Nodes -> Visible -> set to Visible Only
